docs/src/developer/install.md

```diff
@@ -1,36 +1,118 @@
-# Installing miniSASS Django-CMS project
+# Setting up a Local Development Environment
 
-This is currently a basic Django-CMS site. No extra Django applications have been added yet. 
-To install the pre-requisites needed for this site to run, do the following:
+## Prerequisites
 
-## Docker Deployments
+- Any Linux distribution for the operating system
+- Docker installed on the local machine
+- Visual Studio Code for the editor (or any preferred)
 
-* Clone the repository
+## Installation Steps
 
+1. **Open the terminal.**
+
+2. **Create a directory:**
+    ```bash
+    mkdir name_of_directory
+    ```
+
+3. **Clone the minisass repository:**
+    ```bash
+    git clone https://github.com/kartoza/miniSASS.git
+    ```
+
+4. **Navigate into the repo directory:**
+    ```bash
+    cd miniSASS
+    ```
+
+5. **Open the directory with VSCode (assuming you have VSCode installed) or open the repository directory in any editor of your choice.**
+
+6. **Optional: Adjust the build context of the S3 mount.**
+    In case of build failure, replace lines 32-36 in the `docker-compose.yml` file with the following:
+    ```yaml
+    mount:
+      image: kartoza/s3mount
+      build:
+        context: .
+        dockerfile: ./deployment/docker-s3-bucket/Dockerfile
+    ```
+
+7. **Build the container:**
     ```bash
-     git clone git@github.com:kartoza/miniSASS.git
-     cd miniSASS
+    docker compose build
     ```
 
-* Copy the `.example.env` file and adjust the environment variables to match your requirements.
+8. **Start the containers:**
+    ```bash
+    docker compose up -d db; sleep 180; docker compose up -d
+    ```
 
+9. **Access the Django container:**
```

Comment:
This step should not be necessary. We must fix the root cause of this not working out of the box. Maybe we can add it as debugging if the application does not start up

Reply:
@NyakudyaA can we not create a separate ticket and branch to resolve that issue and update the documentation when that fix is in place ,for now we can just point Frank to this documentation so that he can setup ?

Reply:
Yes, we can but maybe just add it as a troubleshooting section

Reply:
@NyakudyaA I added troubleshoot section